src/patient-chart/results-summary/send-email-dialog.component.tsx

```diff
@@ -24,63 +24,10 @@ const SendEmailDialog: React.FC<SendEmailDialogProps> = ({
   const { t } = useTranslation();
 
   const [email, setEmail] = useState();
-  const [file, setFile] = useState();
 
   const { patient, isLoading } = usePatient(patientUuid);
 
-  // setEmail()
-
-  // generate pdf
-
-  const sendEmail = async () => {
-    // initialize nodemailer
-    // var transporter = nodemailer.createTransport({
-    //   service: "gmail",
-    //   secure: true,
-    //   tls: {
-    //     rejectUnauthorized: false,
-    //   },
-    //   auth: {
-    //     user: "services.ugandaemr@gmail.com",
-    //     pass: "Admin1234_",
-    //   },
-    // });
-    // // point to the template folder
-    // const handlebarOptions = {
-    //   viewEngine: {
-    //     extname: ".hbs",
-    //     layoutsDir: "views/",
-    //     defaultLayout: "template",
-    //     partialsDir: "views/partials/",
-    //   },
-    //   viewPath: "views/",
-    //   extName: ".hbs",
-    // };
-    // transporter.use("compile", hbs(handlebarOptions));
-    // const mailOptions = {
-    //   from: "services.ugandaemr@gmail.com",
-    //   to: email,
-    //   subject: `Tests results`,
-    //   attachments: [{ filename: "testResults.pdf", path: file }],
-    // };
-    // transporter.sendMail(mailOptions, (error, info) => {
-    //   if (error) {
-    //     showNotification({
-    //       title: `Error in sending results`,
-    //       kind: "error",
-    //       critical: true,
-    //       description: error?.message,
-    //     });
-    //   } else {
-    //     showToast({
-    //       critical: true,
-    //       title: `Sent Test Results`,
-    //       kind: "success",
-    //       description: `Test results sent Successfully`,
-    //     });
-    //   }
-    // });
-  };
+  const sendEmail = async () => {};
```

Comment:
Is this a placeholder for extra logic?

Reply:
yes this is a placeholder that i will pickup when the API has been created